integrations/Klaviyo/browser.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-underscore-dangle */
+import logger from "../../utils/logUtil";
+import ScriptLoader from "../ScriptLoader";
+import { extractCustomFields } from "../../utils/utils";
+
+class Klaviyo {
+  constructor(config) {
+    this.publicApiKey = config.publicApiKey;
+    this.apiKey = config.privateApiKey;
```

Comment:
are we using this apiKey anywhere?

Reply:
sae please check for other config variables too.

Reply:
Removed unused vars, need to remove from config be also